core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java

```diff
@@ -417,7 +432,7 @@ public ContainerisedDockerCompose(List<File> composeFiles, String identifier) {
     }
 
     @Override
-    public void start() {
+    public void invoke() {
```

Comment:
Changed this (internally used) method name, as it's always a blocking operation and `invoke` seems more accurate.